distributed/worker.py

```diff
@@ -1055,7 +1055,7 @@ def _close(self, *args, **kwargs):
             setproctitle("dask-worker [closing]")
 
             teardowns = [
-                plugin.teardown(self)
+                plugin.teardown()
```

Comment:
Why this change?  This will break backwards compatibility (people who have code that uses this feature today will suddenly have their code break).  If it's not critical, I recommend leaving this the way that it was so that we don't have to bother with a deprecation cycle.

Reply:
The idea was to unify with the interface of the `SchulderPlugin`, but you're right, let's not break anything!